patches/chrome-installer-mac-sign_chrome.py.patch

```diff
@@ -1,29 +0,0 @@
-diff --git a/chrome/installer/mac/sign_chrome.py b/chrome/installer/mac/sign_chrome.py
-index b1cb1f4beaca0f61cb4a1dff0039dcc02c2614f2..321a7c4802ed60bbb91fc918f4f33fce746a79b0 100755
---- a/chrome/installer/mac/sign_chrome.py
-+++ b/chrome/installer/mac/sign_chrome.py
-@@ -44,6 +44,7 @@ def create_config(config_args, development):
-     from signing.chromium_config import ChromiumCodeSignConfig
-     config_class = ChromiumCodeSignConfig
- 
-+    """
```

Comment:
this is no longer a problem? cc @mihaiplesa @mbacchi

Reply:
To give context, this code now moved to `config_factory.py` (see patch below). In there we call `GetBraveSigningConfig` in our signing_helper (see below) before the try/catch block. In `GetBraveSigningConfig` I added a static method `is_chrome_branded` which returns `false`. The catch should not be throwing an exception in this case.